### PR TITLE
Split maven project into a multi modules project

### PR DESCRIPTION
**Pull request by sfalquier**

### CODE REVIEW

**Comment by sfalquier** [comment]
@mbarbet Could you have a look to this job failure?
https://travis-ci.org/gisaia/ARLAS-server/jobs/378565742

File: ogc-csw/src/main/java/io/arlas/server/ogc/csw/CSWService.java [hunk 1]
```diff
@@ -69,7 +70,9 @@
 import static io.arlas.server.utils.CheckParams.isBboxLatLonInCorrectRanges;
 
 
-public class CSWService extends CollectionRESTServices {
+public class CSWService {
+
+    public Logger LOGGER = LoggerFactory.getLogger(CSWService.class);
```
Comment (MohamedHamouGisaia) [inline ogc-csw/src/main/java/io/arlas/server/ogc/csw/CSWService.java]: LOGGER is not used

Comment (MohamedHamouGisaia) [reply ogc-csw/src/main/java/io/arlas/server/ogc/csw/CSWService.java]: Sorry, it's actually used

Comment (mbarbet) [reply ogc-csw/src/main/java/io/arlas/server/ogc/csw/CSWService.java]: it's a forgotten, we must remove these uses, it was for debugging

**Pull request merged**